src/current/_includes/releases/v23.1/v23.1.25.md

```diff
@@ -0,0 +1,80 @@
+- Fixed a bug where the [Databases]({% link v24.1/ui-databases-page.md %}) page crashed if the range information was not available. [#127090][#127090]
+
+<h3 id="v23-1-25-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in which the [`DISCARD`]({% link v23.1/alter-table.md %}) statement was disallowed with [`default_transaction_read_only = on`]({% link v23.1/session-variables.md %}). [#127549][#127549]
+- Fixed a bug where [`ALTER DATABASE ... DROP REGION`]({% link v23.1/alter-database.md %}#drop-region) could fail if any tables under the given database have [indexes on expressions]({% link v23.1/expression-indexes.md %}). [#126600][#126600]
+- Fixed a bug when [restoring]({% link v23.1/restore.md %}) a database with a [composite type]({% link v23.1/create-type.md %}#create-a-composite-data-type). [#126847][#126847]
+- Fixed a bug when inputting `public` role as user name for [built-in compatibility functions]({% link v23.1/functions-and-operators.md %}#compatibility-functions), such as `has_database_privilege` and `has_schema_privilege`. [#126853][#126853]
+- Fixed a bug where CockroachDB could incorrectly evaluate an [`IS NOT NULL`]({% link v23.1/null-handling.md %}#nulls-and-simple-comparisons) filter if it was applied to non-`NULL` tuples that had `NULL` elements, such as `(1, NULL)` or `(NULL, NULL)`. This bug has existed since v20.2.[#126942][#126942]
+- Fixed a bug where [`CREATE TABLE`]({% link v23.1/create-table.md %}) with [index expressions]({% link v23.1/expression-indexes.md %}) could hit undefined column errors on [transaction retries]({% link v23.1/transaction-retry-error-reference.md %}). [#126200][#126200]
+- Fixed a bug that caused a memory leak when executing SQL statements with comments, for example, `SELECT /* comment */ 1;`. Memory owned by a SQL session would continue to grow as these types of statements were executed. The memory would only be released when closing the [SQL session]({% link v24.2/show-sessions.md %}). This bug has been present since v23.1. [#127757][#127757]
+- Fixed a bug where [`CREATE INDEX IF NOT EXISTS`]({% link v23.1/create-index.md %}) would not correctly short-circuit if the given index already existed. [#128239][#128239]
+- Fixed a bug in syntax validation, in which the `DESCENDING` clause was not allowed for non-terminal columns of an [inverted index]({% link v23.1/inverted-indexes.md %}). Only the last column of an inverted index should be prevented from being `DESCENDING`. This is now properly checked. [#128239][#128239]
+- Fixed a bug where an [index]({% link v23.1/indexes.md %}) could store a column in the primary index if that column had a mixed-case name. [#128239][#128239]
+- Fixed a memory leak that could occur when specifying a non-existent virtual cluster name in the connection string. [#128110][#128110]
```

Comment:
are we not linking to the virtual cluster stuff?  fine if true, idk what status of that is

Reply:
Yes, it's good point. We don't have vc stuff in v23.1 as it was private preview at that point. However, I have left this in the notes because there are maybe users on v23.1 using/testing the pcr that this could be useful for.